src/components/RoundSelection/RoundSelection.js

```diff
@@ -1,10 +1,65 @@
-import React from 'react';
+import React, { useContext, useEffect } from 'react';
 import './RoundSelection.css';
 import { Option } from '../Option/Option';
+import { AccessTokenContext } from '../../hooks/TokenContext';
+
+const baseURL = process.env.REACT_APP_SPOTIFY_BASE_URL;
 
 const RoundSelection = (props) => {
-    const {setRoute, setRounds} = props;
+    const {setRoute, setRounds, setAlbums} = props;
+    const accessToken = useContext(AccessTokenContext);
     
+    useEffect(() => {
+        const fetchRandomAlbums = async(accessToken) => {
+
+        const queryParam = 'one%'; // to be randomized
+        const offsetParam = 100; // a number between 1 and 2000
+        const typeParam = 'album';
+        const limitParam = 5; // initial batch
+
+        const queryParams = new URLSearchParams({
+            q: queryParam,
+            type: typeParam,
+            limit: limitParam,
+            offset: offsetParam
+          });
+
+        const stringifiedQueryParams = queryParams.toString();
+
+        // append params to baseURL
+        const searchEndpoint = `${baseURL}/search?` + stringifiedQueryParams;
+
+        const albums = await fetch(searchEndpoint, {
+            headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+            'Authorization': 'Bearer ' + accessToken.token
+            }
```

Comment:
I tihnk you can actually pass the query params in directly:

```ts
const searchEndpoint = `${baseURL}/search`;

const albums = await fetch(searchEndpoint, {
  headers: {
    'Accept': 'application/json',
    'Content-Type': 'application/json',
    'Authorization': 'Bearer ' + accessToken.token
  },
  body: queryParams
})
```

Reply:
This was my thought as well but since it's a GET request we can't pass the params in the body

```
TypeError: Failed to execute 'fetch' on 'Window': Request with GET/HEAD method cannot have body.
```

Reply:
ah okay!